deploy.doc.sh

```diff
@@ -29,6 +29,40 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
 # SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
+check_versions() {
+  packageVersion=$(ruby -rjson -e "j = JSON.parse(File.read('package.json')); puts j['version']")
```

Comment:
We don't want to depend on ruby for the build process.

Normally, during the build process, there should be an environment variable that gets created (`$PACKAGE_JSON_VERSION`). You can try it locally by doing `source read.version.sh` at the top of the repo, then `echo $PACKAGE_JSON_VERSION` to see what it returns.

You should be able to use that env variable in that script also.